### PR TITLE
[Merged by Bors] - ci: Update ci.yml

### PR DESCRIPTION
**Pull request by morenol**
only run integration tests on staging

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -62,6 +62,7 @@ jobs:
 
   integration_tests:
     name: Integration testing on - ${{ matrix.os }}.
```
Comment (sehz) [inline .github/workflows/ci.yml:64]: This should be named cloud integration test? @galibey in that case why k3 is needed?

**Review by sehz — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (morenol) [reply .github/workflows/ci.yml:64]: @sehz these tests run locally. For cloud tests we usually create another workflow and run on intervals

Comment (sehz) [reply .github/workflows/ci.yml:64]: Then cloud login should be removed? This should be strictly run in locally

**Pull request closed without merging**